media-sound/deadbeef/files/deadbeef-1.9.5-clang-strict.patch

```diff
@@ -0,0 +1,17 @@
+https://bugs.gentoo.org/908414
+https://github.com/DeaDBeeF-Player/deadbeef/issues/2987
+
+Fix the build with clang, upstream needs a more involved fix.
+
+--- a/configure.ac
++++ b/configure.ac
+@@ -16,7 +16,8 @@ AC_PROG_INSTALL
+ dnl AC_PROG_LIBTOOL
+ AC_CONFIG_MACRO_DIR([m4])
+ AC_C_BIGENDIAN
+-AM_GNU_GETTEXT
++AM_GNU_GETTEXT([external])
++AM_GNU_GETTEXT_REQUIRE_VERSION([0.22])
```

Comment:
It needs 0.22 for the build issues with clang.

Reply:
It doesn't.  The line before fixes it for 0.21 too.

I replaced it locally, I get this:
```
emerge -1avk =sys-devel/gettext-0.21.1
ebuild deadbeef-1.9.5-r3.ebuild clean prepare
head -1 /var/tmp/portage/media-sound/deadbeef-1.9.5-r3/work/deadbeef-1.9.5/m4/gettext.m4
# gettext.m4 serial 72 (gettext-0.21.1)
emerge -1avk =sys-devel/gettext-0.22-r1
ebuild deadbeef-1.9.5-r3.ebuild clean prepare
head -1 /var/tmp/portage/media-sound/deadbeef-1.9.5-r3/work/deadbeef-1.9.5/m4/gettext.m4
# gettext.m4 serial 77 (gettext-0.22)
```

So `eautopoint` installs the latest gettext it can find, be it 0.21 or 0.22, and all goes well.